doc/examples/reconst_ivim.py

```diff
@@ -0,0 +1,193 @@
+"""
+============================================================
+Intravoxel incoherent motion
+============================================================
+The intravoxel incoherent motion (IVIM) model describes diffusion
+and perfusion in the signal acquired with diffusion. The IVIM model
+can be understood as an adaptation of Stejskals and Tanners work (ref)
+on biologic tissue, and was proposed by Le Bihan et al (ref).
+The hypothesis is that two compartments exist: a slow moving compartment,
+where particles diffuse in a Brownian fashion as a consequence of thermal
+energy, and a fast moving compartment (the vascular compartment), where
+blood moves as a consequence of pressure gradient. In this second
+compartment, a pseudo diffusion term (D* ) is introduced that describes
+on a macroscopic level the displacement of the blood elements in an
```

Comment:
=> "...that describes the displacement of the blood elements in an assumed randomly laid out vascular network, at the macroscopic level"